### ISSUE TITLE
Run polling in a loop that doesn't get stopped by user input

### ISSUE THREAD

**Issue by agent462** [philosowaffle/peloton-to-garmin#91]
Love the tool but I'd love to see a small enhancement with polling.

The current behavior with polling runs in a loop but gets stopped by "Press the <ENTER> key to continue...".  It stops here until obviously you hit enter and it does another loop.

Desired behavior: Using polling you are not stopped by this user input step.  Polling runs on interval specified, does what it needs to do and then sleeps until the next interval.

![Screenshot 2021-04-14 123508](https://user-images.githubusercontent.com/242283/114754161-df3f1d80-9d1d-11eb-807f-6b5041b5726b.png)

**Comment by philosowaffle** [comment]
In your `config.ini` file set [PauseOnFinish](https://github.com/philosowaffle/peloton-to-garmin/blob/master/config.ini#L36) to false.  And let me know if that resolves the issue, otherwise we may have a bug if that flag is not being correctly honored.

**Comment by philosowaffle** [comment]
You're correct. In the non-polling scenario the application normally closes when done which is not usually desired.
